.github/workflows/glibc_fault.yml

```diff
@@ -0,0 +1,182 @@
+name: GLIBC fault test
```

Comment:
If we can't use reusable workflows, could you add a comment at the top here to outline how this workflow differs from the main one and which test targets should be added to both?